apps/src/templates/SchoolInfoInputs.jsx

```diff
@@ -52,6 +54,43 @@ export default class SchoolInfoInputs extends Component {
     },
   };
 
+  constructor() {
+    super();
+    this.bindSchoolNotFound = this.bindSchoolNotFound.bind(this);
```

Comment:
note: you can also declare the method as an arrow function below on line 90 and then this explicit binding won't be needed. e.g.

```jsx
bindSchoolNotFound = (snf) => {
  this.schoolNotFound = snf;
}
```

or as an expression:
```jsx
bindSchoolNotFound = (snf) => this.schoolNotFound = snf;
```

Either way is fine.